atlp-rwanda/rca-barefoot-team6-bn
# ft:User Registration and Email Verification

#### What does this PR do?
Register user and verify that user by sending an email
 ...

#### Description of  the task to be completed?
Implementing User Registration and Email Verification
...

#### How should this be manually tested?
Checkout to ft-register-user-confirm-184638320 and Use Postman to create user with this route: ${Basic URL}/api/users
Get that user with "${Basic URL}/api/users" to view his token
Paste the token in this route "${Basic URL}/api/users/verify-email/${token}" or click the link sent in the email to verify email
...

#### Any background context you want to provide?
Create app password from your google account to get password to be used as APP_PASS env variable
Ensure also that you have EMAIL variable in env file
Then you will be able to send emails to the created users
...

#### What are the relevant pivotal tracker stories?
#184638320, #184638371
...

#### Screenshots (if appropriate)

...
